tests/monotouch-test/AudioUnit/AudioUnitTest.cs

```diff
@@ -53,7 +53,11 @@ public void GetElementCount ()
 		[Test]
 		public void CopyIconTest ()
 		{ 
+#if __MACOS__
+			TestRuntime.AssertXcodeVersion (12, 2);
```

Comment:
The whole point of `AssertXcodeVersion` was to avoid adding `#if`s everywhere...

and don't think this is needed though, wouldn't it be enough to change `AssertXcodeVersion` to map macOS 10.16/11.0 to Xcode 12.2?

Reply:
I agree, but it does not really reflect the truth specially if you compare it with CheckExactXcodeVersion or anywhere else, this is very likely the only time we’ll need to do it so my take is we can live with some extra if/defs this time in favor of consistency